controllers/configuration_controller.go

```diff
@@ -116,7 +117,26 @@ func (r *ConfigurationReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	fmt.Println("HELLO", configuration)
+
 	meta := initTFConfigurationMeta(req, configuration)
+	if r.JobNamespace != "" {
+		uid := string(configuration.GetUID())
+		// @step: since we are using a single namespace to run these, we must ensure the names
+		// are unique across the namespace
+		meta.ApplyJobName = uid + "-" + string(TerraformApply)
```

Comment:
Can you also insert the Configuration name in the ApplyJobNam? It will help link the Configuration and the job at first glance.

Reply:
Added the `configuration.Name` as a prefix to Apply and Destory jobs